essentials/teleport.dsc

```diff
@@ -0,0 +1,210 @@
+teleport_command:
+  type: command
+  name: teleport
+  debug: false
+  description: Teleports you or another player to a location, to a world, or to another player
+  usage: /teleport help / (player) ((x y z (world)) / (world) / (player))
```

Comment:
`/teleport [help] / [<&gt>player<&lt>] (<&gt>x<&lt> <&gt>y<&lt> <&gt>z<&lt>) [<&gt>world<&lt>/<&gt>player<&lt>]`
equivalent to
`/teleport [help] / [<player>] (<x> <y> <z>) [<world>/<player>]`